src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -514,6 +528,27 @@ internal NamedTypeSymbol UnderlyingTupleType
 
         internal bool HasFriendlyNames => _hasFriendlyNames;
 
+        public override ImmutableArray<ITypeSymbol> TupleElementTypes
+        {
+            get
+            {
+                return _fields.SelectAsArray(f => (ITypeSymbol)f.Type);
```

Comment:
I don't think this method ever returns an empty array, there are always fields in a tuple type.